packages/material-ui-codemod/src/v5.0.0/drop-jss-styles.test.js

```diff
@@ -0,0 +1,226 @@
+import fs from 'fs';
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from './drop-jss-styles';
+import readFile from '../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@material-ui/codemod', () => {
+  describe('v5.0.0', () => {
+    describe('drop-jss-styles', () => {
```

Comment:
"drop" could suggest that the codemod only removes jss. Would it be clearer for developers with?

```suggestion
    describe('jss-to-styled', () => {
```

Reply:
I'm good with that.